tests/test_community.py

```diff
@@ -234,9 +234,9 @@ def test_community_leave_post(client):
         )
 
         assert (
-            b'{"access_token":"RefreshToken","message":"Community Left","refresh_token":"AccessToken","status_code":200}\n'
+            b'{"access_token":"RefreshToken","data":{"Members":[{"email":"Testemail@gmail.com","name":"Jeffery","token":"Toke"}],"name":"mock"},"message":"Community Left","refresh_token":"AccessToken","status_code":200}\n'
```

Comment:
the members contains the entire user object? not just the user email (id)?

Reply:
From what it seems it stores email name and token. It should just have the email so I'll update that

Reply:
awesome thanks